proxy/inherited_storage/contracts/UpgradeabilityProxy.sol

```diff
@@ -13,7 +13,7 @@ contract UpgradeabilityProxy is Proxy, UpgradeabilityStorage {
   /**
   * @dev Constructor function
   */
-  function UpgradeabilityProxy(string _version) public {
+  constructor (string _version) public {
     registry = IRegistry(msg.sender);
     upgradeTo(_version);
```

Comment:
What the difference between following two calling ways:
upgradeTo(_version); // msg.sender in upgradeTo()

this.upgradeTo(_version);  // msg.sender  in upgradeTo()

Reply:
Its ok here.